atmos.yaml

```diff
@@ -120,14 +120,48 @@ commands:
     commands:
       - name: hello
         description: This command says Hello world
+        verbose: true
         steps:
           - echo Saying Hello world...
           - echo Hello world
       - name: ping
         description: This command plays ping-pong
+        verbose: false
         steps:
           - echo Playing ping-pong...
           - echo pong
+  - name: show
+    description: Execute 'show' commands
+    # subcommands
+    commands:
+      # If a custom command accepts `component` argument and `stack` flag, atmos generates the component config in the stack
```

Comment:
The only criticism I have is this becomes a magical side effect when you include a specially named argument and flag, otherwise, execution of the side effect silently doesn't happen.  Having these side effects can make managing intended behavior in the future difficult.

Reply:
@RobertHorrox I don't love it myself, but I look at it like this: the component name and stack name are part of the component configuration, so if you don't specify them, then there is no component configuration.